### PR TITLE
Automate Telegram #160

### PR DESCRIPTION
**Pull request by Namyalg**
### Your checklist for this pull request
:warning: Please review the [guidelines for contributing](https://github.com/sanscript-tech/hacking-tools-scripts/blob/main/CONTRIBUTING.md) to this repository.

Parent issue: #160 

### Type of change

**Select all that applies**:
* [ ] New feature (non-breaking change which adds functionality)
* [ ] Bug fix (non-breaking change which fixes an issue)
* [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
* [ ] This change requires a documentation update

### Checklist:

* [x] My code follows the style guidelines(Clean Code) of this project.
* [x] I have performed a self-review of my own code.
* [x] I have used consistent code conventions and clear function/method/variable names.
* [x] I have commented on my code, particularly in hard-to-understand areas.
* [x] I have created a helpful and easy to understand README.md which contains description, requirements and use case.
* [x] I have added relevent screenshots/images if necessary in a seperate folder.
* [x] I have added my work in respective language folder.
* [x] My changes generate no new warnings.

### CODE REVIEW

**Review by tejan-singh — CHANGES_REQUESTED** [review]
you have sent two scripts in same request. Please send them seperately.

**Comment by Namyalg** [comment]
I am unable to identify the script other than the one for Telegram Automation, can you please point it out @tejan-singh

File: Python/Location-based-on-IP/get.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from modulefinder import ModuleFinder
+
```
Comment (tejan-singh) [inline Python/Location-based-on-IP/get.py]: This is different issues I guess.

File: Python/Location-based-on-IP/get.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+finder = ModuleFinder()
+finder.run_script('get_coordinates.py')
```
Comment (tejan-singh) [inline Python/Location-based-on-IP/get.py]: This is also different issue

**Review by tejan-singh — CHANGES_REQUESTED** [review]

File: Python/Location-based-on-IP/get.py (hunk 2 above)
Comment (Namyalg) [reply Python/Location-based-on-IP/get.py]: Okay, I will remove that folder, thanks for the assistance

Comment (Namyalg) [reply Python/Location-based-on-IP/get.py]: I have removed the files

**Pull request merged**